### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by mjpdejong**
Addition of a option (--help) which explains the user how to use the script!

### CODE REVIEW

File: aurBuild.sh [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
-# usage: aurBuild [--install] [package]
+if [[ "$1" == "--help" ]]; 
+then
+	cat <<-HelpMessage
+
+		Author: Sjon Hortensius <sjon@hortensius.net>
+
+		Build a package from AUR. Extra options will be passed to makepkg.
+
+		If no arguments are appied packages which are currently not up-to-date will be displayed.
```
Comment (SjonHortensius) [inline aurBuild.sh]: appied > supplied

**Comment by SjonHortensius** [comment]
Can you also provide a commit-message that explains the change?

File: aurBuild.sh [hunk 2]
```diff
@@ -1,7 +1,21 @@
 #!/bin/bash
-# Author: Sjon Hortensius <sjon@hortensius.net>
-# build a package from aur. Extra options will be passed to makepkg
-# usage: aurBuild [--install] [package]
+if [[ "$1" == "--help" ]];
```
Comment (SjonHortensius) [inline aurBuild.sh]: ';' is an alternative for a newline; eg `if [[ $a ]]; then echo $a; fi`. But you have a newline here so it's redundant

Comment (mjpdejong) [inline aurBuild.sh]: True.
Usually I end the first row of my if statement with ';then', but for the sake of consistency (with your style) I entered the 'then' to the next row but forgot to remove the semicolon.

**Pull request merged**